guides/content_management_systems/TYPO3.md

```diff
@@ -0,0 +1,33 @@
+# TYPO3
+[TYPO3](https://www.typo3.org) is a open source content management system. It is founded in 1998 from Kasper Skarhoj. TYPO3 is written in PHP to configure the sytem TYPO3 has it own language called TypoScript. The system has advantage in multisite websites. Functions can be added with extensions. In this time there are over 5000 Extension in the [offical repository] (https://extensions.typo3.org). The system is designed for multilingualism and is maintained by a user and developer community from around the world.
+
+# Why TYPO3
```

Comment:
You have stated some statistics for TYPO3 and WordPress but nothing much about them. WHY would I switch to WordPress or TYPO3 for content management?
Please refer to articles included in the guide to see how they can be written effectively. :smile: